app/server.go

```diff
@@ -505,7 +507,8 @@ func (s *Server) Start() error {
 	}
 	s.ListenAddr = listener.Addr().(*net.TCPAddr)
 
-	mlog.Info(fmt.Sprintf("Server is listening on %v", listener.Addr().String()))
+	logListeningPort := fmt.Sprintf("Server is listening on %v", listener.Addr().String())
+	mlog.Info(logListeningPort)
```

Comment:
We usually use `Starting X` and then `X stared` in our logs. What do you think about adopting this pattern and using:
```suggestion
	mlog.Info("Started Server", mlog.String("port", listener.Addr().String())
```

Reply:
Maybe also cut the three dots from https://github.com/mattermost/mattermost-server/pull/13370/files#diff-fb51175d7d8419ee63208a0e76554ab3R443

Reply:
I didn't converted it because they were informative messages during the start-up and are thought for being human readable.

Reply:
Keeping here the human readable and added the structured logging at the same time.